apps/storybook/src/DataCurve.stories.tsx

```diff
@@ -26,44 +28,41 @@ const Template: Story<DataCurveProps> = (args) => {
 };
 
 export const Default = Template.bind({});
-Default.args = {
-  abscissas: range(0, mockValues.oneD.length),
-  ordinates: mockValues.oneD,
-  color: 'green',
-};
-Default.argTypes = {
-  showErrors: { control: false },
```

Comment:
I changed the args declaration to be more in line with your previous Storybook changes.

I also made a separate story for changing the `Color`. It is not that important but it pleases me to have one story for one arg